docs/_documentations/project-actions.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title: Project actions
+description: Project actions
+keywords: run, open, import, show, restart, edit, build, logs, tools, project actions, attach, build, disable, enable, validate, refresh
+duration: 1 minute
+permalink: project-actions
+type: document
+order: 30
+parent: 
+---
+
+# Project actions
```

Comment:
Fix up the order of items to match the order in the menu or make them alphabetical.

![image](https://user-images.githubusercontent.com/9484320/75254202-fe8f8a00-57ad-11ea-8f26-cdf0e9041a8b.png)

Reply:
Fixed them to be in the order of the menu and have put an extra line in between sections; let's see how it renders.